ledger.go

```diff
@@ -1554,97 +1018,6 @@ func (l *Ledger) LogChanges(snapshot *avl.Tree, lastBlockIndex uint64) {
 	})
 }
 
-func (l *Ledger) SyncStatus() string {
```

Comment:
I figure we need to clean up the lifecycle of a node a bit - for the time being I temporarily removed it. Made a card on Trello so that we can target this for v0.4.